oci/config/convert/runtime.go

```diff
@@ -123,31 +126,23 @@ func MutateRuntimeSpec(g rgen.Generator, rootfs string, image ispec.Image, manif
 
 	g.SetProcessUID(uint32(execUser.Uid))
 	g.SetProcessGID(uint32(execUser.Gid))
-	g.ClearProcessAdditionalGids()
```

Comment:
Will fix. My intention of always using and relying on those defaults is that those should be meaningful (ideally). In case they would be changed in way that they break something, we have a specific case to point at.